components/SearchResult.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div class="flex flex-col">
+        <div class="result-header flex flex-row">
+            <img alt="profile icon" class="profile-icon" src="@/assets/images/profileIcon.svg" width="25"
+                height="25" />
+            <span class="flex-1 w-4 font-bold pl-2"> {{ name }} </span>
+        </div>
+        <div class="result-details flex flex-row my-2">
+            <span class="pr-2"> {{ specialty }}</span>
+            <span class="self-center">·</span>
+            <span class="px-2"> {{ facilityName }}</span>
+        </div>
+        <div class="result-tags flex flex-row flex-wrap w-64 mb-2">
```

Comment:
Nit: Won't the flex-wrap cause the text items to overflow? I think we should leave it out and allow them to wrap normally like they do by default.

Reply:
it will cause the flex items to wrap instead of overflowing to the right. it's different than text-wrap. 

My thinking is that if we have more than 3 tags, we'll want them to start wrapping into a second row. 

Here's an example
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap

Reply:
Ah, I mistakenly thought this div was the tag itself and not the container!